packages/desktop-client/src/components/reports/graphs/DonutGraph.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { VictoryLabel, VictoryPie, VictoryTooltip } from 'victory';
+
+//import theme from '../chart-theme';
```

Comment:
Why is this not using the chart theme?

Reply:
Done.

Reply:
It wasn't actually changing anything so I had to add a "pie" secion in the theme file and set style variables. All set!